src/components/Login/Login.js

```diff
@@ -17,35 +25,41 @@ function Signin({ isOpen, onClose, onSubmit, onSignupClick }) {
                     aria-label="close"
                     className="popup__close-button-signin"
                 />
-                <form name="signin" className="popup__form" onSubmit={onSubmit}>
+                <form name="signin" className="popup__form" onSubmit={handleLogin}>
                     <h3 className="popup__form-title">Sign in</h3>
                     < fieldset className="popup__form-fieldset" >
                         <label htmlFor="email" className="popup__form-label">Email</label>
                         <input
+                            name="email"
                             type="email"
-                            id="email-input"
                             className="popup__form-input"
                             placeholder="Enter email"
+                            value={values.email}
+                            onInput={handleChange}
+                            autoComplete="on"
                             required
                         />
-                        <span id="email-input-error" className="popup__form-error"></span>
+                        {!isValid && <span id="email-input-error" className="popup__form-error">{errors.email}</span>}
```

Comment:
The error messages are not visible. You need to show them under the inputs

Reply:
Done